releasenotes/notes/default-transpile-properties-610c0f16ec967f34.yaml

```diff
@@ -0,0 +1,13 @@
+---
+features_transpiler:
+  - |
+    Custom subclasses of :class:`.BasePassManager` can now modify their
+    :attr:`~.BasePassManager.property_set` attribute during their :meth:`~.BasePassManager._passmanager_frontend`
+    method, to seed initial properties.  This provides symmetry, as it was previously only possible
+    to read the final properties during :meth:`~.BasePassManager._passmanager_backend`.
+upgrade_transpiler:
```

Comment:
We can argue about whether this is a feature or an upgrade note in the final release round up. I'm not sure pass manager users/pass authors will need to upgrade their code as a result of this change. I'd maybe view it more as a feature because now these fields are always available to any pass. But it's not worth blocking this over.